server/pbench/bin/index-pbench

```diff
@@ -8,30 +8,31 @@
 
 from __future__ import print_function
 
-import sys, os, time, re, stat, copy
-import hashlib, json, glob, csv, tarfile
+import sys, os, time, re, stat, copy, math
+import hashlib, json, glob, csv, tarfile, logging
 
+from random import SystemRandom
+from urllib3 import exceptions as ul_excs, Timeout
+from datetime import datetime, timedelta
+from collections import Counter, deque
+from optparse import OptionParser, make_option
```

Comment:
We probably should go to argparse - optparse is deprecated.

Reply:
Filed issue #886.